config/config.go

```diff
@@ -40,9 +41,10 @@ func Get() (*Config, error) {
 		KafkaVersion:               "1.0.2",
 		KafkaOffsetOldest:          true,
 		KafkaNumWorkers:            1,
-		HelloCalledGroup:           "dp-search-data-importer",
-		HelloCalledTopic:           "hello-called",
-		OutputFilePath:             "/tmp/helloworld.txt",
+		PublishedContentGroup:      "dp-search-data-importer",
+		PublishedContentTopic:      "search-data-import",
+		BatchSize:                  4,
```

Comment:
I think the default should be set to 500 - this will likely make your unit tests fail, but I have made a suggestion to resolve it

Reply:
nope, if we do not get any messages for a wait-time, it  just process the events already in the batch., so unit tests fine.